frontend/src/app/shared/components/confirm/confirm.component.html

```diff
@@ -0,0 +1,5 @@
+<mat-dialog-content><div>{{data.header}}</div></mat-dialog-content>
```

Comment:
Can you also add optional content?
Like header "Are you sure?"
Content: "You are about to delete your user, are you sure you want to do it"